c7n/resources/ec2.py

```diff
@@ -475,7 +475,8 @@ def process_volume_set(self, resource):
             tags = [
                 {'Key': 'Name', 'Value': volume_id},
                 {'Key': 'InstanceId', 'Value': resource['InstanceId']},
-                {'Key': 'DeviceName', 'Value': block_device['DeviceName']}
+                {'Key': 'DeviceName', 'Value': block_device['DeviceName']},
+                {'Key': 'custodian_snapshot', 'Value': ['true']}
```

Comment:
i would prefer a different value here, 'true' would need to be quoted/escaped in yaml as its a keyword for boolean which just makes matching more confusing. ie. think about what the policy looks like to match against these. Also you've made this a list when its a scalar value.